packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.tsx

```diff
@@ -17,11 +16,10 @@ export class PrimaryButton extends BaseComponent<IButtonProps, {}> {
     const { theme, styles } = this.props;
 
     return (
-      <BaseButton
+      <DefaultButton
         { ...this.props }
-        variantClassName='ms-Button--primary'
```

Comment:
if you use <PrimaryButton> it calls default button and says "use the primary version". in essence, it's the same thing as <DefaultButton primary={true}/>
One thing to remember is that we only say DefaultButton because 'Button' is already being used. Each button has a Standard and a Primary variation.